frontend/views/containers/chatroom/image-viewer/PreviewImageArea.vue

```diff
@@ -0,0 +1,375 @@
+<template lang="pug">
+.c-image-view-area(
+  @wheel.prevent.stop='wheelEventHandler'
+)
+  img.c-preview-image(ref='previewImg'
```

Comment:
For accessibility, `img` elements need an `alt` attribute

Reply:
good catch